### ISSUE TITLE
doesn't run when deftest is wrapped

### ISSUE THREAD

**Issue by blinsay** [pjstadig/humane-test-output#6]
I just added a macro that wrapped `deftest` to my tests, and my tests went back to regular clojure.test output. help!

my macro is nothing too fancy - it looks something like this:

``` clojure
(defmacro mytest
  [test-name expected thingy]
  `(deftest ~test-name
     (is (= ~expected
            (myns/somefn ~thingy)))))
```

**Comment by pjstadig** [comment]
The problem is that when you write a macro like this, the `=` function gets fully qualified, and all of the formatting mechanisms are defined on the unqualified symbol. I have released 0.7.1 which also defines the formatting mechanisms on the qualified `clojure.core/=` symbol.

**Comment by blinsay** [comment]
awesome! thank you!
